Client/src/Common/Headline.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+	export let text: string;
+</script>
+
+<div class="text-5xl text-center">{text}</div>
+
+<style>
+	div {
+		margin-bottom: 1rem;
```

Comment:
I think that "Common" folder name is a little confusing, maybe change it to "CommonComponents" or at least place it under "Cmponents".
And what type of components will be in the "Common" folder?

Reply:
I really want to keep tose root folder names short - so I think of like 'Commons' for components and 'Utils' for functions, I'll change to commons though

Reply:
Then maybe "Commons" just shouldn't be at root?
they are also Components

Reply:
I'll add Components/App to application pages structure and Components/commons to commons

Reply:
moved